### PR TITLE
docs(CONTRIBUTING): complete creators guidelines with relator terms

### PR DESCRIPTION
**Pull request by davorpa**
## What does this PR do?
Add info | Improve repo

### Why is this valuable (or not)?

- Completes the creators guidelines with the relator terms introduced/sugested in https://github.com/EbookFoundation/free-programming-books/pull/7034#discussion_r954014539
- Extends those guidelines with other useful sugestions

Related with:
- EbookFoundation/free-programming-books-parser#13

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Used an informative name for this pull request.
- [x] Define the section text
- [ ] Translate into Deutch
- [ ] Translate into Greek
- [ ] Translate into Persian
- [ ] Translate into Filipino
- [ ] Translate into French
- [ ] Translate into Indonesian
- [x] Translate into Italian
- [ ] Translate into Korean
- [ ] Translate into Portuguese
- [ ] Translate into Russian
- [x] Translate into Spanish: @davorpa 
- [ ] Translate into Vietnamese
- [ ] Translate into Chinese (simplified)
- [ ] Translate into Chinese (traditional)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by davorpa** [comment]
### NOTE

Needs text wording/translation reviewing

**Comment by davorpa** [comment]
### NOTE

**Section text is stable. Translations accepted**

**Comment by eshellman** [comment]
I think we should merge what we have and use the branch for contributions

**Comment by davorpa** [comment]
> I think we should merge what we have and use the branch for contributions

Well, go ahead then! @LuigiImVector, please, remember translate to italian when you have a while :wink:

**Comment by eshellman** [comment]
I would merge it without 2e90dd9a21954230e089f598d1742181b407d8e7  Not sure the best way to expose it

File: docs/CONTRIBUTING-it.md [hunk 1]
```diff
@@ -180,7 +180,15 @@ I nostri elenchi forniscono un set minimo di metadati: titoli, URLs, autori, pia
...
-- Per i lavori tradotti, l'autore originale dovrebbe essere incluso.
+- Per i lavori tradotti, l'autore originale dovrebbe essere incluso. We recommend using [MARC relators](https://loc.gov/marc/relators/relaterm.html) to credit creators other than authors, as in this example:
+
+    ```markdown
+    * [A Translated Book](http://example.com/book-it.html) - John Doe, `trl.:` Mike The Translator
+    ```
+
+    here, the annotation `trl.:` uses the MARC relator code for "translator".
+- Use a comma `,` to delimit each item in the author list.
+- You can shorten author lists with "`et al.`".
```
Comment (LuigiImVector) [inline docs/CONTRIBUTING-it.md:191]:
```suggestion
- Per i lavori tradotti, l'autore originale dovrebbe essere incluso. Consigliamo di usare [MARC relators](https://loc.gov/marc/relators/relaterm.html) per accreditare autori diversi dal creatore, per esempio:

    ```markdown
    * [Un libro tradotto](http://example.com/book-it.html) - John Doe, `trl.:` Mike The Translator
    ```

    qui, l'annotazione `trl.:` usa il codice MARC relativo ai "traduttori".
- Usa l virgola `,` per separare ogni elemento nella lista degli autori
- Puoi accorciare la lista degli autori scrivendo "`et al.`" (e altri).
```

**Comment by davorpa** [comment]
> I would merge it without [2e90dd9](https://github.com/EbookFoundation/free-programming-books/commit/2e90dd9a21954230e089f598d1742181b407d8e7) Not sure the best way to expose it

@eshellman, I would prefer it to be merged as it is now. At least now all documents have the same text, translated or not. This has already been done in the past.

Also, if some text is not translated, at least it can be read in English in the same document and in this way, any of the collaborators can translate it at a glance if they want, seeing that there are some missing pieces.

Any inconvenient?

**Pull request merged**